### PR TITLE
Update Bikeshed metadata: add editors, TR link, drop level

### PR DESCRIPTION
**Pull request by tidoust**
This adds spec editors, as discussed in #817, adds the link to the published TR draft, and drops the notion of level so that the spec gets published as `webdriver-bidi` instead of `webdriver-bidi-1` (levels can be introduced later on if that turns out to be needed).

PR to be merged after publication as First Public Working Draft, the TR link won't exist before that.

### CODE REVIEW

**Review by OrKoN — APPROVED** [review]

**Review by jgraham — APPROVED** [review]

**Comment by whimboo** [comment]
As @jgraham mentioned on Matrix we can merge this PR once we are on TR/.

**Comment by tidoust** [comment]
> As @jgraham mentioned on Matrix we can merge this PR once we are on TR/.

Oh, we already are ;)
Spec was published as First Public Working Draft last week, see: https://www.w3.org/TR/webdriver-bidi/

(That publication is the reason why a [call for exclusions](https://lists.w3.org/Archives/Public/public-browser-tools-testing/2024OctDec/0009.html) was sent to the mailing-list)

Once this PR and #818 are merged, I'll setup automatic publication.

**Pull request merged**
